code/src/java/pcgen/core/PlayerCharacter.java

```diff
@@ -3269,14 +3269,14 @@ public void setHasMadeKitSelectionForAgeSet(final int index, final boolean arg)
 
 	public PCLevelInfo getLevelInfo(int index)
 	{
-		return levelInfoFacet.get(id, index);
+		return levelInfoFacet.get(id, index).get();
```

Comment:
This makes no sense.  Some callers of getLevelInfo are gating the result on null, why does the get appear in this method?